policyengine_us/parameters/gov/irs/vita/eligibility/income_limit.yaml

```diff
@@ -0,0 +1,9 @@
+description: IRS's Volunteer Income Tax Assistance (VITA) sites offer free tax help to people who make income no more than this amount.
+values:
+  2021-01-01: 64_000
+metadata:
+  unit: currency-USD
+  label: VITA Program Income Limit
+  reference:
+    - title: Free tax return preparation for qualifying taxpayers
```

Comment:
we should see if we can find more official references - especially for start dates and values in 2021 / 2022 / 2023